ci/prebuilt/BUILD

```diff
@@ -24,8 +24,9 @@ cc_library(
 )
 
 cc_library(
+    # TODO: Remove 'event_pthreads' once no BUIlD files use it
     name = "event_pthreads",
-    srcs = ["thirdparty_build/lib/libevent_pthreads.a"],
+    srcs = glob(["thirdparty_build/lib/libevent_pthreads.a"]),
```

Comment:
Why glob?

Reply:
See context above, ninja build doesn't produce `libevent_pthreads.a` and all symbols are included in `libevent.a`, but the before the CI image update it have to include `libevent_pthreads.a` from prebuilt, `glob` here is a hack to make CI passes before #3909.

Reply:
I think a comment explaining this reasoning would then be appropriate.